lib/core/src/sdk.rs

```diff
@@ -340,6 +316,50 @@ impl LiquidSdk {
         Ok(())
     }
 
+    async fn start_swapper_tasks(self: &Arc<LiquidSdk>) {
+        let cloned = self.clone();
+        tokio::spawn(async move {
+            let maybe_swapper_proxy_url =
+                match BreezServer::new("https://bs1.breez.technology:443".into(), None) {
+                    Ok(breez_server) => {
+                        let maybe_swapper_proxy_url = breez_server
+                            .fetch_boltz_swapper_urls()
+                            .await
+                            .ok()
+                            .and_then(|swapper_urls| swapper_urls.first().cloned());
+                        if let Some(swapper_proxy_url) = maybe_swapper_proxy_url.clone() {
+                            cloned
+                                .persister
+                                .set_swapper_proxy_url(swapper_proxy_url)
+                                .unwrap_or_else(|err| {
+                                    error!("Could not set cached swapper proxy url: {err:?}")
+                                });
+                        }
+                        maybe_swapper_proxy_url
+                    }
+                    Err(_) => cloned.persister.get_swapper_proxy_url().unwrap_or(None),
+                };
+
+            if let Err(err) = cloned.swapper.connect(maybe_swapper_proxy_url.clone()) {
```

Comment:
The problem here is that by the time the swapper connects it is not available and users may attempt to use the sdk right after initialization receiving an error.
I think we can solve that by passing a closure or trait implementation for fetching the swapper url and let the swapper use that on demand (when creating the client). This is the only way I see we can defer the fetch of the url and maintaining the swapper immediate availability.